src/frontend/components/injector-tree/injector-tree.ts

```diff
@@ -93,6 +102,8 @@ export class InjectorTree implements OnChanges {
 
     let posX, posY, x1, y1, x2, y2;
 
+    const nodesToDraw = [];
```

Comment:
@rajinder-yadav Not sure I agree here. To me, using let would suggest to that the variable is expected to be reassigned at some point in this block. My thinking is that devs should understand that not being able to reassign the variable of course does not mean the assigned value can't be mutated. Your way would suggest that const is only used for assignment of immutable values?